lib/Tvm.js

```diff
@@ -270,7 +269,7 @@ class Tvm {
     logger.info('start of request Id - ' + requestId)
 
     if (params.metricsUrl) {
-      setMetricsURL(params.metricsUrl)
+      setMetricsURL(params.metricsUrl + METRICS_KEY)
```

Comment:
do we need have defensive code here in case metricsUrl param does not end with a backslash?

Reply:
example: https://gist.github.com/shazron/cafb12db9e69f5e68901b82cf1a63272